src/main/webapp/resources/js/pages/projects/redux/templatesSlice.js

```diff
@@ -108,6 +144,9 @@ export const templatesSlice = createSlice({
   },
 });
 
-export const { setTemplate } = templatesSlice.actions;
+export const {
+  setTemplate,
```

Comment:
What is `setTemplate`?

Reply:
Some text that was accidentally added to this export. Removed in 8e63a642f3f46bf750d3af026924c0835221c272